src/library/scala/util/matching/Regex.scala

```diff
@@ -110,6 +110,13 @@ import java.util.regex.{ Pattern, Matcher }
  *  val allYears = for (m <- date.findAllMatchIn(dates)) yield m.group(1)
  *  }}}
  *
+ *  To check if a string matches the entire regex:
```

Comment:
`To check whether the entire input is matched by the regex`. "Input" is more general term than string, and "entire input" instead of "entire regex".